src/shared/components/Settings/Profile/Work/index.jsx

```diff
@@ -6,6 +6,7 @@
 /* eslint-disable no-nested-ternary */
 /* eslint-disable jsx-a11y/label-has-for */
 /* eslint-disable no-undef */
+/* eslint-disable react/jsx-boolean-value */
```

Comment:
Instead add `eslint-disable react/jsx-boolean-value`, please update:

`joinValues={true}`
to
`joinValues`